internal/cmd/gocdk/serve_test.go

```diff
@@ -24,47 +24,22 @@ import (
 )
 
 func TestBuildForServe(t *testing.T) {
-	t.Run("NoWire", func(t *testing.T) {
-		testBuildForServe(t, map[string]string{
-			"main.go": `package main
+	// TODO(light): This test is not hermetic because it brings
```

Comment:
This TODO is not accurate: it only brought in an external module for the Wire case. Please remove.

Reply:
Ah, right -- done.